mods/cnc/maps/funpark01/scj01ea.lua

```diff
@@ -84,7 +84,7 @@ WorldLoaded = function()
 	Trigger.OnIdle(DinoTrex, DinoTrex.Hunt)
 
 	ReinforceWithLandingCraft(RifleReinforcments, SeaEntryA.Location, BeachReinforceA.Location, BeachReinforceA.Location)
-	Trigger.AfterDelay(DateTime.Seconds(1), function() InitialUnitsArrived = true end)
+	Trigger.AfterDelay(DateTime.Seconds(3), function() InitialUnitsArrived = true end)
```

Comment:
The boolean controls when we start to check for living player units. For some reason the first units are added to world later than one second into the game. I have to admit that I don't know the exact cause of this, we're at bisecting at the moment.
> 00:17:17 (pchote) abcdefg30: what's the deal with the dino mission?
00:17:27 (pchote) why does that fix it?
00:17:50 (pchote) and what regressed it in the first place?
00:17:55 (abcdefg30) if that boolean is set to true we check if the player has any units which are Required
00:18:04 (abcdefg30) I don't know what regressed, that's the scary bit
00:18:10 (abcdefg30) something with unloading maybe
00:18:33 (pchote) we really need to find this out
00:18:40 (abcdefg30) the transport unloads units, and after those are added to the world, we want to check if the player has any more units remaining
00:18:41 (pchote) because other missions have probably broken too
00:18:52 (abcdefg30) meh, I already thought you'll say that
00:18:55 (abcdefg30) ^^
00:19:03 (pchote) because you know i'm right :P
00:19:03 (abcdefg30) what about git blaming it?
00:19:07 (abcdefg30) yeah
00:19:16 (abcdefg30) either way, the 3 seconds won't hurt
00:19:26 (pchote) I agree